modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -66,3 +66,32 @@ Defaults to 4 seconds.
 The presence ping gather job parameter changes the timeout setting for gathering the presence ping, in seconds.
 Adjust the [systemitem]``java.salt_presence_ping_gather_job_timeout`` parameter.
 Defaults to 1 second.
+
+
+== Disabling the Salt Mine
+
+In older versions, {productname} used a tool called Salt mine to check minion availability.
+The Salt mine would cause minions to ping the server every hour, which created significant load.
+With the introduction of a more efficient mechanism in {productname}{nbsp}3.2, the Salt mine is no longer required.
+Instead, the {productname} server uses Taskomatic to ping only the minions that appear to have been offline for twelve hours or more, with all minions being pinged at least once in every twenty four hour period by default.
```

Comment:
Do we want to describe how to change that default, or link it from here?

Reply:
I can't immediately see where we've documented this, so maybe we should write it up.

Reply:
It's the `web.system_checkin_threshold` parameter in `rhn.conf`, expressed in days, default value is 1.